### ISSUE TITLE
Missing styles. Is the correct theme chosen for this layout?

### ISSUE THREAD

**Issue by Wolftein** [negusoft/holoaccent#36]
Android Studio can't preview any theme that reference to HoloAccent Theme.

> Rendering Problems 
> 
> Missing styles. Is the correct theme chosen for this layout?  Use the Theme combo box above the layout to choose a different layout, or fix the theme style references.  Failed to find style 'textViewStyle' in current theme (16 similar errors not shown)

```
<style name="Theme.MyTheme" parent="Theme.HoloAccent.Light.DarkActionBar">
</style>
```

**Comment by blurkidi** [comment]
No, the layout designer can't preview the HoloAccent themes. There is a lot of "magic" going on at runtime to make coloring possible, and it won't work in the designer. I recommend using a default Holo theme for designing, the end result should be the same with HoloAccent but with the color applied.

**Comment by Wolftein** [comment]
If add HoloAccent as a module instead of a library (Gradle), then it would work and display everything correctly, can this be related to Android Studio?

**Comment by blurkidi** [comment]
It _would_ work or it _does_ work? Did you try it? Please let us know if you found out something.

**Comment by Wolftein** [comment]
Android Studio preview it if i add HoloAccent as module; if i add it as gradle dependency then it won't work.

**Comment by AnselC** [comment]
Create a new workspace.

**Comment by YKarkouri** [comment]
How to fix that?

**Comment by ramkotirapido** [comment]
How to fix this issue?

**Comment by azadatc** [comment]
I create Template projects in android studio and it simply fails to render the page in preview. 
Keeps spitting out that the component can't be found in theme.

This only happens on my windows machine and not on my Mac

**Comment by MomduhNasser** [comment]
i have issue when creat empty project 
message was (Failed to find style 'coordinatorLayoutStyle' in current theme ) 
i use 3.1.3 for Windows 64-bit (758 MB)
 
![default](https://user-images.githubusercontent.com/16746978/43379936-a43aec98-93d7-11e8-94ec-b60f96322105.png)

**Comment by omarreess** [comment]
same problame . . please any one find the solution
